### PR TITLE
Add logo to iOS and Android projects

### PR DESCRIPTION
**Pull request by vebjornhaugland**
Add icons to iOS and Android application to improve cosmetics of application.

## Changes :balloon:

Modified the icon files of the iOS and Android projects. The application now displays the icon designed and created by @JonJohansen.

In the Android project both the /drawable and /mipmap directories are populated with the new icons with suitable sizes. The iOS icons are modified through the /Asset Catalogs/Assets file.

UWP icons have not been added and thus is still stuck with the default Xamarin icon.

## Types of changes

What types of changes does your code introduce? _Mark boxes that apply with an `x`._

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist :heavy_check_mark:

_Mark boxes that apply with an `x`. These can also be filled out after the pull request is created._

- [ ] I have read the coding convention [guidelines](https://github.com/Dualog-students/dualog-students.github.io#company-guidelines) and my code follows these.
- [ ] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

No further comments. I rest my case.

### CODE REVIEW

File: Diplomatic.Android/Diplomatic.Android.csproj [hunk 1]
```diff
@@ -86,18 +86,24 @@
...
     <AndroidResource Include="Resources\drawable\waltdisney.png" />
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Resources\drawable-hdpi\" />
-    <Folder Include="Resources\drawable-xhdpi\" />
-    <Folder Include="Resources\drawable-xxhdpi\" />
-    <Folder Include="Resources\drawable-xxxhdpi\" />
-    <Folder Include="Resources\drawable\" />
-  </ItemGroup>
+  <ItemGroup />
```
Comment (MaxJohansen) [inline Diplomatic.Android/Diplomatic.Android.csproj]: I had one of these empty ItemGroups show up in one of my commits too: I have no idea what causes it, but they can safely be removed.

**Review by MaxJohansen — DISMISSED** [review]

File: Diplomatic.Android/Diplomatic.Android.csproj (hunk 1 above)
Comment (IsakSundeSingh) [reply Diplomatic.Android/Diplomatic.Android.csproj]: Do eet <3

**Review by IsakSundeSingh — DISMISSED** [review]
Nice

**Review by IsakSundeSingh — APPROVED** [review]
Better now

**Pull request merged**